e2e-tests/insurance/cypress/e2e/journeys/export-contract/agent-service/validation/agent-service-validation.spec.js

```diff
@@ -0,0 +1,104 @@
+import { field as fieldSelector, noRadioInput } from '../../../../../../../pages/shared';
+import { MAXIMUM_CHARACTERS } from '../../../../../../../constants';
+import FIELD_IDS from '../../../../../../../constants/field-ids/insurance/export-contract';
+import { INSURANCE_ROUTES } from '../../../../../../../constants/routes/insurance';
+import { ERROR_MESSAGES } from '../../../../../../../content-strings';
+
+const {
+  ROOT: INSURANCE_ROOT,
+  EXPORT_CONTRACT: { AGENT_SERVICE },
+} = INSURANCE_ROUTES;
+
+const {
+  AGENT_SERVICE: { IS_CHARGING, SERVICE_DESCRIPTION },
+} = FIELD_IDS;
+
+const {
+  INSURANCE: {
+    EXPORT_CONTRACT: {
+      AGENT_SERVICE: AGENT_SERVICE_ERROR_MESSAGES,
+    },
+  },
+} = ERROR_MESSAGES;
+
+const expectedErrorsCount = 2;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context('Insurance - Export contract - Agent service page - As an Exporter, I want to give details about how the agent helped me win the export contract, So that UKEF can understand the working relationship between myself and the agent.', () => {
```

Comment:
should this be Agent service page - Validation?

Reply:
great spot, have updated